pkg/server/helpers.go

```diff
@@ -443,6 +340,7 @@ func getRuntimeOptions(c containers.Container) (interface{}, error) {
 
 const (
 	// unknownExitCode is the exit code when exit reason is unknown.
+	// TODO(windows): Figure out unknown exit status for windows.
 	unknownExitCode = 255
```

Comment:
Runtime V2 defined this as 255 so we return that for Windows as well

Reply:
Nice. Sound good.